src/StylesProvider.js

```diff
@@ -8,7 +8,6 @@ import themeLight from './styles/themeLight';
 
 const themeNameToObject = {
   'default': themeLight,
-  'light': themeLight,
```

Comment:
I didn't get that commit. The app is working fine after removing this. Tested it on both iOS and android.